src/pages/Payment/Payment.tsx

```
@@ -70,54 +58,40 @@
-            <span className="w-20 font-medium">예금주명:</span>
-            <span>윤성민</span>
-          </div>
+      {(payInfo && payInfo.status === "PENDING") ?
+        <Information
+        senderName={senderName}
+        isLoading={isLoading}
+        payInfo={payInfo}
+        remainingAmount={remainingAmount}
+        /> : 
+        <Alert className="bg-green-50 border-green-200 text-green-800 shadow-sm">
+        <CircleCheckBig size={24} color="#166534"  />
```

Biome (1.9.4)

[error] These CSS classes should be sorted.

Safe fix: Sort the classes.


(lint/nursery/useSortedClasses)